tools/rna_tools/ribotaper/conda/r-multitaper/build.sh

```diff
@@ -0,0 +1,17 @@
+
+#!/bin/bash
+
+# R refuses to build packages that mark themselves as
```

Comment:
conda package will be removed from the final version. the branch is still in progress.